### ISSUE TITLE
T8 Rocket still uncraftable

### ISSUE THREAD

**Issue by bartimaeusnek** [GTNewHorizons/GT-New-Horizons-Modpack#4735]
Due to the need of T8 materials in the UHV Assembler, you cant craft the T8 rocket. 
Possible Solutions:

1. Re-Tier the T8 Rocket to UV
2. Re-Tier the UHV Components to T7 Materials
3. Add the needed Materials to T7 Planets

**Comment by BlackFlameTNT** [comment]
or change the cable to bedrock cable that would work.

**Comment by yukieiji** [comment]
I agree with 3
I think that add small ore  of CosmeticNeuronium and Bedrokium to T7 planet and increase probability of BlackPlutoniumOre which is material of CosmeticNeutronium.

**Comment by BlackFlameTNT** [comment]
I disagree with adding cosmic neutroium and bedrock at T7 the problem is with infinity and awaken thingy are needed for UHV superconductors you can make bedrock from mv and u can get cosmic neutroium not cheap but it is doable

**Comment by Dream-Master** [comment]
@bartimaeusnek @yukieiji @BlackFlameTNT why T8 Rockets not craftable? Details please.

**Comment by BlackFlameTNT** [comment]
Assembler UHV need 2 UHV superconductors to be crafted look at UHV mechanic hull UHV superconductors can be change
UHV superconductors or
All UHV hulls and stuff to use bedrock instead of superconductors or
Change the T8 plate to UV.

**Comment by BlackFlameTNT** [comment]
I would recommend changing the superconductors UHV base make it use nether star and bedrock dust instead of awaken and infinity you will still need to craft UHV pump to make it superconductors and change the tiny pipe to nether star

**Comment by ScriptedPiky** [comment]
Picture of the UHV hull recipe by the way:
![2019-06-21_17 40 36](https://user-images.githubusercontent.com/41557212/59957298-ba443100-944b-11e9-8178-f88af251bcd5.png)

Recommended to change UHV superconductor cables in the recipe to bedrockium cables (possibly 8x or 12x cables?). Otherwise, you will need to make UHV superconductors possible in a UV assembly machine instead of UHV, and changing many many other parts of the recipes as well to make UHV hulls possible.

Basically a run-down:
-Infinity Catalyst is not obtainable in T6 (UV) or even in T7 for that matter (UHV)
-Awakened Draconium is not obtainable in T6 (UV), due to the requirement of the main quest to visit Pluto (T7/UHV)
-The T7 rocket requires a UHV Assembler, which is gated behind the UHV Machine Hull using UHV superconductor that requires said tiered Assembler

**Comment by nanmumc** [comment]
Similar problems occurred while trying to make Motor(UHV). It is unable to collect the needed materials.

**Comment by nanmumc** [comment]
OK，you can use 4 energy hatches to forced boost the voltage to UHV for GT++ Assembler. I am sure this is not the right way, but it works.

**Comment by draknyte1** [comment]
Considering the cost, that seems like a fair solution. 😂

**Comment by Cinobi** [comment]
100% craftable, just push ur GT++ assembler to UHV with 4 UV hatches.

**Comment by Prometheus0000** [comment]
It's craftable now
